sceptre/cli/dump.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import click
+
+from sceptre.context import SceptreContext
+from sceptre.cli.helpers import (
+    catch_exceptions,
+    write
+)
+from sceptre.plan.plan import SceptrePlan
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(name="dump")
+def dump_group():
+    """
+    Commands for dumping attributes of stacks.
+    """
+    pass
+
+
+@dump_group.command(name="config")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def dump_config(ctx, path):
+    """
+    Dump the rendered (post-Jinja) Stack Configs as yaml.
```

Comment:
It's not just yaml. If output format is set to json, it outputs as json.

Reply:
Ok, deleted 'as yaml'.